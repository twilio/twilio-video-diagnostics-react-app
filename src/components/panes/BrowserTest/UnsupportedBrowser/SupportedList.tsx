import { makeStyles, Typography, Table, TableHead, TableBody, TableRow, TableCell, Link } from '@material-ui/core';

const useStyles = makeStyles({
  tableHead: {
    background: '#F4F4F6',
  },
  table: {
    '& td': {
      padding: '0.9em',
    },
  },
});

export function SupportedList() {
  const classes = useStyles();

  return (
    <>
      <Table className={classes.table}>
        <TableHead className={classes.tableHead}>
          <TableRow>
            <TableCell>
              <Typography variant="body1">
                <strong>Supported browsers</strong>
              </Typography>
            </TableCell>
          </TableRow>
        </TableHead>
        <TableBody>
          <TableRow>
            <TableCell>
              <Typography variant="body1">
                <Link
                  color="inherit"
                  underline="always"
                  target="_blank"
                  rel="noopener"
                  href="https://www.google.com/chrome/"
                >
                  Chrome
                </Link>
              </Typography>
            </TableCell>
          </TableRow>
          <TableRow>
            <TableCell>
              <Typography variant="body1">
                <Link
                  color="inherit"
                  underline="always"
                  target="_blank"
                  rel="noopener"
                  href="https://www.mozilla.org/en-US/firefox/new/"
                >
                  Firefox
                </Link>
              </Typography>
            </TableCell>
          </TableRow>
          <TableRow>
            <TableCell>
              <Typography variant="body1">
                <Link
                  color="inherit"
                  underline="always"
                  target="_blank"
                  rel="noopener"
                  href="https://support.apple.com/downloads/safari"
                >
                  Safari
                </Link>
              </Typography>
            </TableCell>
          </TableRow>
          <TableRow>
            <TableCell style={{ borderBottom: 'none' }}>
              <Typography variant="body1">
                <Link
                  color="inherit"
                  underline="always"
                  target="_blank"
                  rel="noopener"
                  href="https://www.microsoft.com/en-us/edge"
                >
                  Edge
                </Link>
              </Typography>
            </TableCell>
          </TableRow>
        </TableBody>
      </Table>
    </>
  );
}
