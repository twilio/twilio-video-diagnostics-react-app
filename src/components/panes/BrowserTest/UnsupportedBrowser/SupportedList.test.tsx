import { shallow } from 'enzyme';
import { SupportedList } from './SupportedList';

describe('the SupportedList component', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<SupportedList />);

    expect(wrapper).toMatchSnapshot();
  });
});
