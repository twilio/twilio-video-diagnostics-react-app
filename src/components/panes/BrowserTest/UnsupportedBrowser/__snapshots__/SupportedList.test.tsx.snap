// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the SupportedList component should render correctly 1`] = `
<Fragment>
  <WithStyles(ForwardRef(Table))
    className="makeStyles-table-2"
  >
    <WithStyles(ForwardRef(TableHead))
      className="makeStyles-tableHead-1"
    >
      <WithStyles(ForwardRef(TableRow))>
        <WithStyles(ForwardRef(TableCell))>
          <WithStyles(ForwardRef(Typography))
            variant="body1"
          >
            <strong>
              Supported browsers
            </strong>
          </WithStyles(ForwardRef(Typography))>
        </WithStyles(ForwardRef(TableCell))>
      </WithStyles(ForwardRef(TableRow))>
    </WithStyles(ForwardRef(TableHead))>
    <WithStyles(ForwardRef(TableBody))>
      <WithStyles(ForwardRef(TableRow))>
        <WithStyles(ForwardRef(TableCell))>
          <WithStyles(ForwardRef(Typography))
            variant="body1"
          >
            <WithStyles(ForwardRef(Link))
              color="inherit"
              href="https://www.google.com/chrome/"
              rel="noopener"
              target="_blank"
              underline="always"
            >
              Chrome
            </WithStyles(ForwardRef(Link))>
          </WithStyles(ForwardRef(Typography))>
        </WithStyles(ForwardRef(TableCell))>
      </WithStyles(ForwardRef(TableRow))>
      <WithStyles(ForwardRef(TableRow))>
        <WithStyles(ForwardRef(TableCell))>
          <WithStyles(ForwardRef(Typography))
            variant="body1"
          >
            <WithStyles(ForwardRef(Link))
              color="inherit"
              href="https://www.mozilla.org/en-US/firefox/new/"
              rel="noopener"
              target="_blank"
              underline="always"
            >
              Firefox
            </WithStyles(ForwardRef(Link))>
          </WithStyles(ForwardRef(Typography))>
        </WithStyles(ForwardRef(TableCell))>
      </WithStyles(ForwardRef(TableRow))>
      <WithStyles(ForwardRef(TableRow))>
        <WithStyles(ForwardRef(TableCell))>
          <WithStyles(ForwardRef(Typography))
            variant="body1"
          >
            <WithStyles(ForwardRef(Link))
              color="inherit"
              href="https://support.apple.com/downloads/safari"
              rel="noopener"
              target="_blank"
              underline="always"
            >
              Safari
            </WithStyles(ForwardRef(Link))>
          </WithStyles(ForwardRef(Typography))>
        </WithStyles(ForwardRef(TableCell))>
      </WithStyles(ForwardRef(TableRow))>
      <WithStyles(ForwardRef(TableRow))>
        <WithStyles(ForwardRef(TableCell))
          style={
            Object {
              "borderBottom": "none",
            }
          }
        >
          <WithStyles(ForwardRef(Typography))
            variant="body1"
          >
            <WithStyles(ForwardRef(Link))
              color="inherit"
              href="https://www.microsoft.com/en-us/edge"
              rel="noopener"
              target="_blank"
              underline="always"
            >
              Edge
            </WithStyles(ForwardRef(Link))>
          </WithStyles(ForwardRef(Typography))>
        </WithStyles(ForwardRef(TableCell))>
      </WithStyles(ForwardRef(TableRow))>
    </WithStyles(ForwardRef(TableBody))>
  </WithStyles(ForwardRef(Table))>
</Fragment>
`;
